sunpy/map/mapcube.py

```diff
@@ -232,19 +248,28 @@ def peek(self, gamma=None, resample=None, **kwargs):
         Examples
         --------
         >>> cube = sunpy.Map(files, cube=True)
-        >>> ani = cube.plot(colorbar=True)
+        >>> ani = cube.peek(colorbar=True)
         >>> plt.show()
 
         Plot the map at 1/2 original resolution
 
         >>> cube = sunpy.Map(files, cube=True)
-        >>> ani = cube.plot(resample=[0.5, 0.5], colorbar=True)
+        >>> ani = cube.peek(resample=[0.5, 0.5], colorbar=True)
+        >>> plt.show()
+
+        Plot the map with the limb at each time step
+
+        >>> def myplot(fig, ax, map):
+        ...    p = map.draw_limb()
+        ...    return p
+        >>> cube = sunpy.Map(files, cube=True)
+        >>> ani = cube.peek(plot_function=myplot)
```

Comment:
The mapcube.plot() function needs the same example, or a similar one.
Also, the examples should not use the variable name "map", since that is a
Python built-in function.  smap?

Also, mapcube.plot() does not have the interactive controls of map.peek().
Is this intended?

Other than that, I have tried the code and it works.  So, with the small
edits above, the PR gets a +1 from me.

Thanks, Stuart.



On Fri, Jun 12, 2015 at 9:17 AM, Stuart Mumford <notifications@github.com>
wrote:

> In sunpy/map/mapcube.py
> <https://github.com/sunpy/sunpy/pull/1423#discussion_r32314459>:
>
> >          >>> plt.show()
> >
> >          Plot the map at 1/2 original resolution
> >
> >          >>> cube = sunpy.Map(files, cube=True)
> > -        >>> ani = cube.plot(resample=[0.5, 0.5], colorbar=True)
> > +        >>> ani = cube.peek(resample=[0.5, 0.5], colorbar=True)
> > +        >>> plt.show()
> > +
> > +        Plot the map with the limb at each time step
> > +
> > +        >>> def myplot(fig, ax, map):
> > +        ...    p = map.draw_limb()
> > +        ...    return p
> > +        >>> cube = sunpy.Map(files, cube=True)
> > +        >>> ani = cube.peek(plot_function=myplot)
>
> @wafels <https://github.com/wafels> it does now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1423/files#r32314459>.
>